Automattic/woocommerce-payments
# BNPLs: updated methods copy on settings page

Fixes #6799

#### Changes proposed in this Pull Request

- Updated Affirm/Afterpay PMs description on the setting page.

#### Testing instructions

* Check settings page to ensure the copy is updated
* Check JS console to ensure no JS errors are being introduced (incl. when checking/unchecking boxes)

-------------------


**Post merge**
